Rakefile

```diff
@@ -36,4 +36,4 @@ Rake::TestTask.new do |t|
   t.verbose = true
 end
 
-task :default => :test
+task default: [:test, :rubocop]
```

Comment:
I'm actually ok with this change, cause rubocop on my machines runs pretty fast.
If the resistance is that sometimes you just want to run tests, ya can always explicitly run `rake test`